src/qt/createwalletdialog.cpp

```diff
@@ -111,6 +112,28 @@ CreateWalletDialog::~CreateWalletDialog()
     delete ui;
 }
 
+#ifdef ENABLE_EXTERNAL_SIGNER
+void CreateWalletDialog::setSigners(std::vector<ExternalSigner>& signers)
+{
+    if (!signers.empty()) {
+        ui->external_signer_checkbox->setEnabled(true);
+        ui->external_signer_checkbox->setChecked(true);
+        ui->encrypt_wallet_checkbox->setEnabled(false);
+        ui->encrypt_wallet_checkbox->setChecked(false);
+        // The order matters, because connect() is called when toggling a checkbox:
+        ui->blank_wallet_checkbox->setEnabled(false);
+        ui->blank_wallet_checkbox->setChecked(false);
+        ui->disable_privkeys_checkbox->setEnabled(false);
+        ui->disable_privkeys_checkbox->setChecked(true);
+        const std::string label = signers[0].m_name;
```

Comment:
24815c6309431cb0797defaf7add1150bcf4b567

Could add comment "TODO show available signers in combobox, for now use 1st signer name"?

Reply:
@promag it is just the name of the wallet so I don't think it would be worth adding a combobox tbh. That might be confusing because it wouldn't actually change between signers, it would just change the name.